### PR TITLE
Banner template

### PR DESCRIPTION
**Pull request by ludwiczakpawel**
## What type of PR is this? (check all applicable)

- [x] Feature

## Description

One of our clients needs a specific banner so we took this opportunity to create a unified template for other clients too. This PR puts some foundations to how it's gonna look like but it does NOT affect any visuals currently. It's basically bunch of CSS that needs to be ready for whenever someone creates a specific campaign banner.

## QA Instructions, Screenshots, Recordings

Basically just look at the code as there's no real visual change right here.

## Added/updated tests?

- [x] No, not needed

### CODE REVIEW

File: app/assets/stylesheets/components/banners.scss [hunk 1]
```diff
@@ -55,3 +55,43 @@
     text-decoration: underline;
   }
 }
+
+.c-banner {
+  background: var(--banner-bg);
+  color: RGB(var(--banner-color));
```
Comment (ludwiczakpawel) [inline app/assets/stylesheets/components/banners.scss:61]: Uppercase `RGB` because otherwise our sass compiler goes nuts 🤷

**Review by jeremyf — APPROVED** [review]
